pipeline/db/db.go

```diff
@@ -717,15 +717,15 @@ func QueryAll(
 	// 	"FULL OUTER JOIN developers d ON (a.developer = d.id) " +
 	// 	"FULL OUTER JOIN app_hosts h ON (a.id = h.id) " +
 	// 	"FULL OUTER JOIN app_perms p ON (a.id = p.id) " +
-	// 	"FULL OUTER JOIN app_packages pkg  ON (a.id = pkg.id) " +
+	// 	"FULL OUTER JOIN app_packages pkg  ON (a.id = pkg.id) "
 
 	queryStr = "SELECT " +
 		"id, title, summary, description, store_url, price, free, rating, " +
 		"num_reviews, genre, family_genre, min_installs, max_installs, updated, " +
 		"android_ver, content_rating, recent_changes, app, store, region, " +
 		"version, icon, analyzed, email, name, store_site, site, hosts, permissions, " +
 		"packages " +
-		"FROM  all_apps_data " +
+		"FROM  matview.apps_play_data " +
```

Comment:
matview -> mat_view

Reply:
but like its already changed on the database? :L